### ISSUE TITLE
Darkstalkers Chronicle (graphic bug Intel HD)

### ISSUE THREAD

**Issue by brujo5** [hrydgard/ppsspp#12455]
I can't see the characters, the images speak for themselves.

![ULES00016_00000](https://user-images.githubusercontent.com/4172898/67709652-9ce41e00-f97b-11e9-8833-2c9d396abf95.png)
![ULES00016_00001](https://user-images.githubusercontent.com/4172898/67709653-9ce41e00-f97b-11e9-9075-7cd825076750.png)
![ULES00016_00002](https://user-images.githubusercontent.com/4172898/67709654-9ce41e00-f97b-11e9-8d64-f5270148bcdd.png)
![Sin título](https://user-images.githubusercontent.com/4172898/67709682-a9687680-f97b-11e9-948b-089ef75f0467.png)

I'm with windows 7 and the same thing happens regardless of whether I use directx or opengl

lof file 
[ppsspplog.zip](https://github.com/hrydgard/ppsspp/files/3780269/ppsspplog.zip)

**Comment by hrydgard** [comment]
Use the latest build from https://buildbot.orphis.net/ppsspp/ . Unfortunately for some reason it seems the android and 32-bit builds have failed, but you should be able to use the 64-bit Windows build.

**Comment by brujo5** [comment]
I am using the latest version from buildbot.

**Comment by brujo5** [comment]
I'm thinking maybe it's because windows 7 doesn't have full direct11 support. there are extensions that are only available as of windows 8.1 (I think)

**Comment by LunaMoo** [comment]
No software renderer works same on all backends.
 You should update your assets folder as if you only updated exe file, it will not work which I'm guessing happens here. Also with that hardware you probably should never change the game option from it's default "Wide" presentation as your cpu is too weak to run that game without the hack and hack only works for Wide mode..

**Comment by brujo5** [comment]
thanks @LunaMoo 
it work now but only for opengl backend
![Sin título](https://user-images.githubusercontent.com/4172898/67712741-cc962480-f981-11e9-8d24-cafd4901c685.png)
i use windows screen capture because for some reason ppsspp f12 screenhot look black whit this game, the game is going faster but I have other programs running now.

en direct11 is unplayable and run very slow
![Sin título](https://user-images.githubusercontent.com/4172898/67713229-a8871300-f982-11e9-8165-92fb2c77b284.png)

**Comment by brujo5** [comment]
work fine with dirext9 but runs very slow compared to OpenGL

![Sin título](https://user-images.githubusercontent.com/4172898/67713618-67dbc980-f983-11e9-8aaa-5a480f7cd6ab.png)
I have changed settings but I can't make it work with direct11

**Comment by unknownbrackets** [comment]
Hm, maybe something about the RB upload failed?  Slow is one thing, but the graphics being funny is another.

-[Unknown]

**Comment by hrydgard** [comment]
Ah, I know what the D3D11 problem is. D3D11 doesn't support 16-bit formats on Windows 7 and earlier. Easily fixable, I'll take care of it soon.

**Comment by brujo5** [comment]
just update it to windows 10 pro.  the game works fine and without graphic bug with DX11 but why is running slow compared with opengl and DX9. With dx11 it goes to half the speed that runs on dx9

Shouldn't it be the opposite?? thx

**Comment by LunaMoo** [comment]
Nope, it shouldn't be the opposite. In typical case, it should be about the same, since most of the work is being done on the CPU with the GPU only doing the final stretch and display.

 However yours is not typical case, it's a low end 2 core laptop/netbook with intel graphics that's not known for optimized drivers. It would not be a big deal in other scenarios since CPU emulation is rarely if ever a bottleneck in PPSSPP and GPU rendering is possibly faster with d3d on your system, however software renderer needs much more CPU power and when you're bottlenecked by it, a bigger driver overhead between apis would be enough to make a noticeable difference and that's probably what happens.

 In short a smaller driver overhead(GPU drivers make use of your CPU) of selected API would give you more performance in that specific scenario than it's actual rendering performance.

**Comment by hrydgard** [comment]
Reopening this for the 16-bit texture issue on older windows, that's very fixable.

Regarding performance, haven't really optimized the software rendering image submission for speed, I would suspect that texture creation/deletion is a bit expensive on some D3D11 implementations, I don't have any other explanation for why it might be slower.

Regardless, for now, just use whatever backend works the best for you.

**Comment by brujo5** [comment]
Thank you both for your explanation, now I understand what happens.

**Comment by unknownbrackets** [comment]
How does this look now?

-[Unknown]

**Comment by brujo5** [comment]
@unknownbrackets 
ppsspp-v1.9.3-155-g08ca4ee23-windows-amd64

The game run in black screen after game intro.

ppsspp debug log: 
[ppsspp_debug_log.zip](https://github.com/hrydgard/ppsspp/files/4000830/ppsspp_debug_log.zip)

ppsspp info log:
[ppsspp_info_log.zip](https://github.com/hrydgard/ppsspp/files/4000831/ppsspp_info_log.zip)

Merry Christmas!!

**Comment by MojoJojoDojo** [comment]
some people still get this graphics bug even after updating to the latest PPSSPP build:
https://media.discordapp.net/attachments/480103153803460614/681920481854488692/unknown.png
from PPSSPP discord.

**Comment by LunaMoo** [comment]
That's most likely because they updated executable only. Tell them to update assets as well.

**Comment by hrydgard** [comment]
Oh right, compat.ini from assets is required to be up to date.

**Comment by hrydgard** [comment]
Closed because this game works now, with the software-rendered workaround.
